content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -93,10 +91,15 @@ The following apply to every installation:
 
 The system can use either S3 or Azure, so you only need to provide one set of the following variables.
 
+Select which will be used, S3 or Azure:
+
+- `placement` - (Required) Set to `placement_s3` for S3 and `placement_azure` for Azure
+
 For S3:
 
-- `aws_access_key_id` — (Required) AWS access key ID for S3 bucket access. To use AWS instance profiles for this information, set it to `""`.
-- `aws_secret_access_key` — (Required) AWS secret access key for S3 bucket access. To use AWS instance profiles for this information, set it to `""`.
+- `aws_instance_profile` (Optional) When set, use credentials from the AWS instance profile. If selected, `aws_access_key_id` and `aws_secret_access_key` are not required. Set to `1` to use the instance profile. Defaults to `0`.
```

Comment:
I might move the "If selected" sentence to the end here.

Reply:
Not sure what you mean.

Reply:
I think she's saying it should read like:

> When set, use credentials from the AWS instance profile.  Set to `1` to use the instance profile. Defaults to `0`.  If selected, `aws_access_key_id` and `aws_secret_access_key` are not required. 

I agree that reads a little easier.